### PR TITLE
Update: enforce single quotes around strings

### PR DESCRIPTION
**Pull request by blazzy**
https://github.com/stylelint/stylelint/blob/master/src/rules/string-quotes/README.md

### CODE REVIEW

**Comment by rygine** [comment]
LGTM (pending key change)

File: index.js [hunk 1]
```diff
@@ -86,6 +86,7 @@ module.exports = {
     'selector-type-no-unknown': true,
     'shorthand-property-no-redundant-values': true,
     'string-no-newline': true,
+    string: 'single',
```
Comment (rygine) [inline index.js]: `string-quotes` is the key

**Comment by blazzy** [comment]
The doc was really confusing. Didn't realize `string` in the options description was describing the type of the option. `string: "single"|"double"`

Updated.

**Comment by rygine** [comment]
merge when rdy :)

**Comment by paulyoung** [comment]
Squash these commits?

**Comment by blazzy** [comment]
What commits? :-)

**Comment by blazzy** [comment]
Slipped an `-m --amend` in the commit command.

**Pull request merged**